sixtools/GUI_components.py

```diff
@@ -0,0 +1,171 @@
+import pandas as pd
+from collections import OrderedDict
+import datetime
+import matplotlib
+import matplotlib.pyplot as plt
+from cycler import cycle
+from databroker import DataBroker as db
+import ipywidgets
+from traitlets import TraitError
+
+# Set defaults
+pd.set_option('display.height', 500)
+pd.set_option('display.max_rows', 500)
+matplotlib.rcdefaults()
+markers = cycle(['o', 's', '^', 'v', 'p', '<', '>', 'h'])
+
+
+# Functions
+def stopped(header):
```

Comment:
Can we discuss why this stuff under # Functions is not in another module? It may be that they belong here but I suspect they may also be useful in other sections of code and hence perhaps belong in a different module.

Reply:
Right now, I can only envisage these being used by the GUIs.